libs/build_cdo.sh

```diff
@@ -34,7 +34,7 @@ if $MODULES; then
   fi
 else
   prefix=${CDO_ROOT:-"/usr/local"}
-  enable_pnetcdf=$(nc-config --has-pnetcdf)
+  [[ ${DOWNLOAD_ONLY} =~ [yYtT] ]] || enable_pnetcdf=$(nc-config --has-pnetcdf)
```

Comment:
Sorry, I don't understand what this change is for.

Reply:
If DOWNLOAD_ONLY is turned on then `nc-config` isn't available.

Reply:
Understood.

I think we can completely remove this line from here and place it right before we start building with
```
enable_pnetcdf=$(nc-config --has-pnetcdf)
```
It will be available there.